README.md

```diff
@@ -311,6 +311,34 @@ ytch.getChannelCommunityPosts(payload).then((response) => {
  }
  ```
 
+ **getChannelStats(payload)**
+
+Gets the stats of a channel
+- payload (Object) (Required) - An object containing the various options
+  - channelId (String) (Required) - The channel ID to get stats from
+  - channelIdType (Integer) (Optional) - defined as for `getChannelInfo()`
```

Comment:
Maybe easier to read if updated to
```suggestion
  - channelIdType (Integer) (Optional) - Same definition as `channelIdType` in `getChannelInfo()`
```

Same for `httpsAgent`?

However I have just checked `channelIdType` in `getChannelInfo()` description which looks wrong:
> Grabs newest comments when `true`. Grabs top comments when `false`